solvebio/client.py

```diff
@@ -107,12 +107,23 @@ def __init__(self, host=None, token=None, token_type='Token',
         # intermittent connection errors.
         retries = Retry(
             total=5,
+            connect=5,
             backoff_factor=0.1,
             status_forcelist=[
                 codes.bad_gateway,
                 codes.service_unavailable,
                 codes.gateway_timeout
+            ],
+            allowed_methods=frozenset([
```

Comment:
Any thoughts on using an environment variable to include non-idempotent methods to the list, or some kind of "aggressive retry mode" for people going through proxies?

`SOLVEBIO_RETRY_ALL=1` or something to enable PUT/POST/PATCH/DELETE?
We can enable HEAD/GET/OPTIONS/TRACE by default

Reply:
yes we can make it like that - I'll get on it